### PR TITLE
Add changelog 1.0.4.0

### PR DESCRIPTION
**Pull request by NicolasDorier**

### CODE REVIEW

**Comment by NicolasDorier** [comment]
ping @kukks @rockstardev @pavlenex @dennisreimann @bolatovumar there might have missing stuff, let me know

File: Changelog.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+   * Granular permissions
+   * Endpoint for creating a new user
+   * Endpoint for creating API Keys
+   * More details in the documentation
+* Greenfield API C# Client
+
+### Altcoins
+
+* Decimal precision for Liquid assets fixes
+* Add L-CAD support for Liquid
```
Comment (Kukks) [inline Changelog.md:44]: * Small Monero fixes

Comment (NicolasDorier) [reply Changelog.md:44]: more details?

Comment (Kukks) [reply Changelog.md:44]: nothing definite really, just tried to make it a little more stable

**Comment by pavlenex** [comment]
LGTM, nothing missing as far as I can tell.

**Pull request merged**